cliUsage.html

```diff
@@ -118,24 +121,27 @@ <h5>For loading protocol jars other than eiffelsemantics:</h5>
         </p>
 
         <h5>If you want to have the message publishing logs add d flag:</h5>
-        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -mp eiffelsemantics -d</code></pre>
+        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en exchangeName -mb localhost:5672 -mp eiffelsemantics -d</code></pre>
 
         <h5>If you want to have the message publishing on RabbitMQ secured port (5671):</h5>
         <ul>
             <li>tls option is used to run REMReM on RabbitMQ secured port (5671).</li>
             <li>tls option value is either 'default' or &lt;version&gt; for secured port.</li>
         </ul>
-        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -mp eiffelsemantics -tls 1.2</code></pre>
+        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en exchangeName -mb localhost:5672 -mp eiffelsemantics -tls 1.2</code></pre>
 
         <p><strong>NOTE:</strong> for RabbitMQ default port 5672, no need to pass -tls option.</p>
 
         <h5>If you want to have the message publishing on RabbitMQ with given routing key:</h5>
-        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -mp eiffelsemantics -rk myroutingkey</code></pre>
+        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en exchangeName -mb localhost:5672 -mp eiffelsemantics -rk myroutingkey</code></pre>
 
         <h5>If you want to have the message publishing on RabbitMQ with given tag:</h5>
-        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -mp eiffelsemantics -tag production</code></pre>
+        <pre><code>java -jar publish-cli.jar -f publishMessages.json -en exchangeName -mb localhost:5672 -mp eiffelsemantics -tag production</code></pre>
+
+        <h5>If there are no exchanges, then we can create an exchange and publish the messages to it by using create_exchange property. Example:</h5>
```

Comment:
To create an exchange passed while publishing the messages use --create_exchange or -ce to true.